HW0/P5/P5.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import math
+
+x = np.linspace(1, 150, num=50)
+y1 = [int(math.ceil(math.log(i, 2))) for i in x]
+y2 = [i - 1 for i in x]
+y2[0] = 1
+
+plt.plot(x, y1, 'bs')
+plt.plot(x, y2, 'g^')
+plt.xlabel('Number of Bags')
+plt.ylabel('Counting Time (s)')
+plt.title('P5 plot')
+plt.show()

```

Comment:
If you plot this with the y-axis on a log-scale you really get a nice view of how the "infinite" worker problem is characterized.